scripts/spot-grid.py

```diff
@@ -211,16 +216,17 @@ def start_generate(self):
         self.notify(self._detector.array)
 
 
-def live_display(ccd, dataflow, kill_ccd=True, gridsize=None):
+def live_display(ccd, dataflow, kill_ccd=True, gridsize=None, pxsize=PIXEL_SIZE_SAMPLE_PLANE / DEFAULT_MAGNIFICATION):
     """
     Acquire an image from one (or more) dataflow and display it with a spot grid overlay.
     ccd: a camera object
     dataflow_name: name of the dataflow to access
     kill_ccd: True if it is required to terminate the ccd after closing the window
     gridsize: size of the grid of spots.
+    px_size (float): pixelsize in µm
```

Comment:
No, please, follow the convention of values in meters. You can convert to µm in the display.